opensrp-anc/src/test/java/org/smartregister/anc/library/model/LoginModelTest.java

```diff
@@ -43,6 +43,7 @@ public void testIsUserLoggedOutShouldReturnTrue() {
         Assert.assertTrue(model.isUserLoggedOut());
     }
 
+    @Ignore
```

Comment:
Why are you Ignoring tests?

Reply:
this change came under the conflict resolution which is corrected